server/dash/requirements.txt

```diff
@@ -1,22 +1,33 @@
 Brotli==1.0.9
-click==7.1.2
-dash==1.19.0
-dash-core-components==1.15.0
-dash-html-components==1.1.2
-dash-renderer==1.9.0
-dash-table==4.11.2
-Flask==1.1.2
-Flask-Compress==1.8.0
-future==0.18.2
-gunicorn==20.0.4
-itsdangerous==1.1.0
-Jinja2==2.11.3
-MarkupSafe==1.1.1
+click==8.1.2
+colorama==0.4.4
+dash==2.4.1
+dash-core-components==2.0.0
+dash-html-components==2.0.0
+dash-table==5.0.0
+Flask==2.1.1
+Flask-Compress==1.12
+importlib-metadata==4.11.3
+itsdangerous==2.1.2
+Jinja2==3.1.1
+MarkupSafe==2.1.1
+plotly==5.7.0
+tenacity==8.0.1
+Werkzeug==2.1.1
+zipp==3.8.0
+
+# Other alraedy existed
```

Comment:
Are these changes necessary?

Reply:
These updates are necessary to ensure dash and its dependencies are most up to date.